etc/copyright-exclude.txt

```diff
@@ -66,3 +66,5 @@ persistence_3_0.xjb
 persistence_3_0.xsd
 # excluded as this is a test file and we need to validate its content
 src/test/resources/static/classpath/index.html
+.hbs
+._java_
```

Comment:
just decided on that name - feel free to suggest an alternative.  It can't be ".java" because it confuses IDEA and other tooling into thinking it's a real source file in the project.  This is used as a resource for unit testing - comparing what is actually code-generated to what was expected to be code-generated. So I could use the extension ".java.expected" or ".java-expected", etc.

Reply:
just for testing